src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -187,6 +187,15 @@ def validate_load_balancer_sku(namespace):
             raise CLIError("--load-balancer-sku can only be standard or basic")
 
 
+def validate_sku_tier(namespace):
+    """Validates the sku tier string."""
+    if namespace.tier is not None:
+        if namespace.tier == '':
+            return
+        if namespace.tier.lower() != "free" and namespace.tier.lower() != "standard":
+            raise CLIError("--tier can only be free or standard")
```

Comment:
Take `InvalidArgumentValueError` instead of `CLIError`

Reply:
Fixed.